packages/graphql-hooks/src/middlewares/apqMiddleware.ts

```diff
@@ -1,4 +1,5 @@
 import { Sha256 } from '@aws-crypto/sha256-browser'
+import { Buffer } from 'buffer'
```

Comment:
do we need this? it's a fairly large package

Reply:
this is not pollyfilled with webpack 5, so the APQ middleware throws an exception in a browser environment, one option would be to require a `sha256` function as an option to the middleware and let the consumer choose, another option would be to separate the apq middleware to a separate package, which may also provide better encapsulation (currently imported from `graphql-hooks/lib/middlewares/apqMiddleware`)